### PR TITLE
Register resources the plone 5.1 way

### PR DESCRIPTION
**Pull request by busykoala**
Javascipt resources need to be registered differently in plone 5.1.

### CODE REVIEW

**Comment by busykoala** [comment]
@mbaechtold, sorry I didn't wait until tests passed. I assumed that registering the resources would not break them, after the others passed.

I'll put you in here again if I fixed the things.

**Comment by busykoala** [comment]
@mbaechtold, so nun sollte es aber klappen mit den Tests. Könntest du falls dir die Änderungen passen, gerade mergen um dann deinen Branch möglichst bald zu releasen? Ich möchte ihn gerne in ftw.events verwenden 😜

**Comment by mbaechtold** [comment]
@busykoala, until I can do a proper review, you may install this package from your branch `mo/plone5_resource_registry` in `ftw.events`, so you don't have to wait for a release.

**Comment by busykoala** [comment]
I oversaw this, because we usually only have one profile per package. I have trouble registering them in the right order and keep getting errors there.

Regarding usage from branch: I was doing that, but Mäthu wanted me to release this first and use it from Pypi. That's why I started this Branch in the first place.

**Comment by busykoala** [comment]
@maethu do you see, what I do wrong registering the profile here?

**Comment by busykoala** [comment]
I'm not quite sure if I can validate it like I did in https://github.com/4teamwork/ftw.keywordwidget/pull/36/commits/705bb06fcf29562e88b77f2908ea942739d4ac64 but since I can access the resources of ftw.keywordwidget I believe that it must be registered.

**Comment by busykoala** [comment]
@mbaechtold @maethu
🛎

File: ftw/keywordwidget/configure.zcml [hunk 1]
```diff
@@ -74,10 +114,13 @@
 
     <include package="ftw.upgrade" file="meta.zcml" />
     <upgrade-step:directory
+        zcml:condition="not-have plone-5"
```
Comment (maethu) [inline ftw/keywordwidget/configure.zcml]: uhhh what's the reason for that? do we need different upgrade steps for plone 4 and plone 5?

Comment (busykoala) [reply ftw/keywordwidget/configure.zcml]: I removed it no. I added this in a desperate moment when registering wasn't working and forgot to remove it after that.

**Pull request merged**